apps/token/.env.testnet

```diff
@@ -1,7 +1,7 @@
 # App configuration variables
 NX_VEGA_ENV=TESTNET
 NX_VEGA_CONFIG_URL=https://static.vega.xyz/assets/testnet-network.json
-NX_VEGA_URL=https://api.n10.testnet.vega.xyz/graphql
+NX_VEGA_URL=https://api.n08.testnet.vega.xyz/graphql
```

Comment:
(non-blocking): do you need to connect to a specific node by default?

Reply:
also you may need to update name of the PR after last linter updates

Reply:
I believe we need a default node. I think we switched to that one at the time when n10 was down, but it was a fairly arbitrary choice.

Thanks for the heads up on the linter update :)